src/main/java/org/jenkinsci/plugins/buildnamesetter/BuildNameSetter.java

```diff
@@ -5,11 +5,13 @@
 import hudson.matrix.MatrixAggregatable;
 import hudson.matrix.MatrixAggregator;
 import hudson.matrix.MatrixBuild;
+import hudson.matrix.MatrixProject;
```

Comment:
Correction: This change will probably break the plugin for anyone having disabled Matrix Project.